src/models/IBGFeatureRequest.ts

```diff
@@ -0,0 +1,13 @@
+export interface IBGFeatureRequest {
+  /**
+   * the name of featureRequest
+   *
+   */
+  name: string;
+
+  /**
+   * variant is an optional param
+   *
+   */
+  variant?: string;
+}
```

Comment:
I think this should be `IBGFeatureFlag` or `FeatureFlag` instead of `IBGFeatureRequest`. I like the `FeatureFlag` one better TBH as we usually don't use the `IBG` prefix in the React Native SDK unless there's a chance of a conflict. Let's not forget to rename the file as well.

Let's also make the comments more useful, I'm thinking something like this would work.

```suggestion
export interface FeatureFlag {
  /** The name of the feature flag */
  name: string;

  /**
   * The variant of the feature flag.
   * Leave it `undefined` for boolean (kill switch) feature flags.
   */
  variant?: string;
}
```